### DISCUSSION TITLE
Export getTextContent helper

### DISCUSSION THREAD

**Discussion by jvliwanag** [callstack/react-native-testing-library#1626] [Ideas]
Exporting `getTextContent` is particularly useful for this kind of assertion:

```js
expect(findManyRows().map((r) => within(r).findManyCells().map((c) => getTextContent(c))).toStrictEqual([
  ["foo", "FOO"],
  ["bar", "BAR"],
]);
```

Workaround is to loop through everything running expect on every ReactTestInstance encountered, but test errors are harder to decipher.

The dom version of the testing-library already has a precedent [here](https://testing-library.com/docs/dom-testing-library/api-custom-queries#getnodetext).

**Comment by mdjastrzebski** [comment]
@jvliwanag We certainly could export this function. I'll try to squize PR for it in the next couple of days.
